modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/TagSelector.es.js

```diff
@@ -71,17 +50,18 @@ export default ({tagsChange, tagsLoaded, tags = [], ...props}) => {
 	return (
 		<>
 			<ClayForm.Group className="c-mt-4">
-				<label htmlFor="basicInput">
-					{Liferay.Language.get('tags')}
-				</label>
-				<ClayMultiSelect
-					{...props}
-					inputValue={inputValue}
-					items={items}
-					onChange={setInputValue}
-					onItemsChange={filterItems}
-					sourceItems={sourceItems}
-				/>
+				<div className="questions-tag-selector">
+					<AssetTagsSelector
+						eventName={`_${context.portletNamespace}_selectTag`}
+						groupIds={[context.siteKey]}
```

Comment:
siteKey works? I thought it needed the siteId

Reply:
You are right but `context.siteKey` contains the ScopeGroupId. Maybe we should rename `siteKey` to siteId? We should use siteKey in the rest of Questions?

Reply:
Got it!